style.css

```diff
@@ -0,0 +1,42 @@
+nav { display: flex; }
+header { display: inline; }
+#hamburger { width: 18px; }
+#search { width: 18px; }
+#nyt { width: 260px; }
+#sub { background-color: rgba(4, 4, 90, 0.795); }
+#login { background-color: rgba(4, 4, 90, 0.795); }
+#login:hover { background-color: rgba(5, 5, 118, 0.795); }
+#sub:hover { background-color: rgba(5, 5, 118, 0.795); }
+#flex-kid1 { margin-right: 20px; }
+#flex-kid1:hover { text-decoration: underline; }
+#flex-kid2 { margin-right: 20px; }
+#flex-kid2:hover { text-decoration: underline; }
+#flex-kid3 { margin-right: 20px; }
+#flex-kid3:hover { text-decoration: underline; }
+#flex-kid4 { margin-right: 20px; }
+#flex-kid4:hover { text-decoration: underline; }
+#flex-bold { margin-right: 20px; }
+#article { font-family: 'Montserrat', sans-serif; }
+#article h2 { font-size: 30px; }
+#article h3 { font-size: 18px; }
+#icons1 { width: 25px; }
+#icons2 { width: 25px; }
+#icons3 { width: 25px; }
+#icons4 { width: 25px; }
+#icons5 { width: 25px; }
+#icons6 { width: 25px; }
```

Comment:
Always keep your code DRY. Refactor the above rules to the class selector and ensure that there's no repetition. i.e.
```
.icons1, 
.icons2, 
.icons3, 
.icons4, 
.icons5, 
.icons6 { 
  width: 25px; 
}
```

Kindly check through other instances where you're having this behavior in your  CSS rules.